Dockerfile

```diff
@@ -57,6 +57,11 @@ RUN cd ${PROJECT_PATH}
 # Export benchmarks results as a volume
 VOLUME ${BENCHMARKS_OUTPUT_PATH}
 
+# Add script to test TCP connections of external services that are specified
+# as dependencies for the runner container
+COPY wait-for.sh /usr/local/bin
+RUN ln -s usr/local/bin/wait-for.sh / # backwards compat
```

Comment:
what is the backwards compatible part here? I know it's somewhere else, I just don't know docker and wonder